packages/gatsby-cli/src/init-starter.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface InitOptions {
+  rootPath?: string
```

Comment:
this would make this file empty right? is it worth keeping `InitOptions` in there anyway as it's TS?

Reply:
And this is a broader question that I wasn't sure how to address. I simply took the definition files from flow annotations and placed them into `.d.ts` files with the same name. If we're wanting to only keep the definitions, I'm not sure this is the right way to do them (as, like you'd mentioned, the TS language extensions MIGHT (?) assume that these files are empty), but I'm not sure how else we'd want to handle these files. 

That said, I've not noticed any differences with IDE behavior (including Ctrl+Click) in either Webstorm or VS Code so I'm unsure if this is the behavior which occurs.

Reply:
so flow types and typescript are not the same, they have slightly different syntax

Reply:
Well and there are various implementation differences. I've written a bit of both. A `.d.ts` file is just a definition file, though. It's not a typescript logic file. All the `.d.ts`s are doing in this PR is enabling the JSDoc comments to be able to pick up the formed type for IDE TS Language extensions (built into TS itself, provided by MS for other IDEs even outside of vs code)

That said, I could continue writing the definition files to include the actual shapes of the file's exports if that's what we want, but I feel they're out-of-scope for this issue itself